src/components/_global/BalRangeInput/BalRangeInput.vue

```diff
@@ -4,7 +4,7 @@ import 'vue-slider-component/theme/antd.css';
 import { computed } from 'vue';
 import VueSlider, { DefineComponent as TVueSlider } from 'vue-slider-component';
 
-import { theme } from '@/../tailwind.config';
+import tailwindConfig from '@/../tailwind.config';
```

Comment:
Why does destructuring an import not work here?

Reply:
I made many changes to make tailwind work with vite and forgot this one after making it work. Fixed 👍